src/libs/ComponentSpecs.js

```diff
@@ -0,0 +1,512 @@
+/**
```

Comment:
This file includes all the i13n implementation. and expose a pickSpec function.
so that we can get the functions we need to create the HOC or the mixins.